src/main/php/rdbms/tds/TdsV5Protocol.class.php

```diff
@@ -269,6 +269,9 @@ public function query($sql) {
       } else if ("\xE3" === $token) {   // ENVCHANGE, e.g. from "use [db]" queries
         $this->envchange();
         return null;
+      } else if ("\x79" === $token) {   // RETURN_STATUS (eg. from stored procedures)
+        $result = $this->stream->getLong();
+        $token= $this->stream->getToken();
```

Comment:
The most efficient way of discarding would be `$this->stream->read(4);`. For documentation reasons, you could leave the `getLong()` call there, but the assignment is definitively not needed:)